cypress/automated-tests/youth-pass/successful-youth-pass-submission.spec.js

```diff
@@ -0,0 +1,65 @@
+const faker = require('faker');
+const constants = require('../../common/youth-pass-constants');
+
+describe('Youth Pass Successful Submission', () => {
+    it('Submits a Youth Pass Application', () => {
+        const youthPassUrl = constants.getYouthPassConstants().youthPassUrl;
+        const applicantFirstName = faker.name.firstName();
+        const applicantLastName = faker.name.lastName();
+        const applicantEmailAddress = `${applicantFirstName}${applicantLastName}_${faker.datatype.number()}@example.com`;
```

Comment:
While `faker` does have email address functionality, it uses real domains. I think it's safer to avoid this by only using `example.com` addresses.